fossunited/foss_hackathon/doctype/foss_hackathon_localhost/test_foss_hackathon_localhost.py

```diff
@@ -0,0 +1,169 @@
+import frappe
+import frappe.permissions
+from frappe.tests import IntegrationTestCase
+
+from fossunited.doctype_ids import (
+    HACKATHON_LOCALHOST,
+    USER_PROFILE,
+)
+from fossunited.tests.utils import insert_test_hackathon, insert_test_hackathon_localhost
+
+
+class TestFOSSHackathonLocalHost(IntegrationTestCase):
+    def setUp(self):
+        self.ORGANIZER_1 = "test1@example.com"
+        self.organizer1_profile = frappe.get_doc(USER_PROFILE, {"user": self.ORGANIZER_1})
+        self.ORGANIZER_2 = "test2@example.com"
+        self.organizer2_profile = frappe.get_doc(USER_PROFILE, {"user": self.ORGANIZER_2})
+        self.hackathon = insert_test_hackathon()
+        self.localhost = insert_test_hackathon_localhost(parent_hackathon=self.hackathon.name)
+
+    def tearDown(self):
+        frappe.set_user("Administrator")
+        frappe.delete_doc(HACKATHON_LOCALHOST, self.localhost.name, force=True)
+        self.hackathon.delete(force=True)
```

Comment:
nit - delete the localhost too?
i guess we don't really need to worry about the profiles as long as remove the relevant roles from them, which happens during localhost deletion automatically, right?

Reply:
ah, line 23 is supposed to be like 24, forgot to change